packages/grid/x-data-grid-premium/src/DataGridPremium/DataGridPremium.tsx

```diff
@@ -1008,7 +1008,7 @@ DataGridPremiumRaw.propTypes = {
   /**
    * If `true`, the grid will not use `valueFormatter` when exporting to CSV or copying to clipboard.
    * If an object is provided, you can choose to ignore the `valueFormatter` for CSV export or clipboard export.
-   * @default: false
+   * @default false
```

Comment:
This is technically touching non-MIT code.
Do we need to go with the SLA process for even non-functional changes?
cc @joserodolfofreitas @DanailH

Reply:
@LukasTy I believe we need a CLA for *any* change in the commercially licensed folder.

Reply:
Although in our [CLA docs](https://www.notion.so/mui-org/CLA-Contributor-License-Agreement-92ece655b1584b10b00e4de9e67eedb0#50f662641b8048cb969f1ad400d04983) it says:

> If the change is nontrivial and user-facing, the CLA needs to be signed.

Reply:
I have updated https://www.notion.so/mui-org/CLA-Contributor-License-Agreement-92ece655b1584b10b00e4de9e67eedb0?pvs=4#a1f27812396a4cfd9a7891e61fb2acd7 to detail this more, inspired by OpenSSL. Here, I think we can consider it's a trivial change, if the author agrees too, we could move forward without CLA

Reply:
Yeah, sure, let's move forward